packages/docs/core-concepts/index.md

```diff
@@ -63,3 +63,22 @@ export default defineComponent({
   },
 })
 ```
+
+You can use `storeToRefs` to destructure the returned object without losing reactivity:
```

Comment:
```suggestion
In order to extract properties from the store while keeping its reactivity, you need to use `storeToRefs()`. It will create refs for any reactive property. This is useful when you are only using state from the store but not calling any action:
```